_components/step-indicator/guidance/usability.md

```diff
@@ -1,5 +1,5 @@
 - **Indicate where a user is in the process.** Use a color and text treatment for the current step segment that is distinct from both the completed and pending step segments. The current segment should be the most visually prominent segment in the step indicator. Pending segments should be the least visually prominent of the segments, but should still maintain accessible contrast and not appear disabled.
 - **Use short or single-word labels.** If labelling the step indicators, stick to very short words.
-- **Provide navigation separately.** Use navigation like a [button group](){{ site.baseurl }}/components/button-group to step forward and backward through a form.
+- **Provide navigation separately.** Use navigation like a [button group](https://designsystem.digital.gov/components/button-group/) to step forward and backward through a form.
```

Comment:
We should generally use `{{ site.baseurl }}` in place of `https://designsystem.digital.gov` 

```suggestion
- **Provide navigation separately.** Use navigation like a [button group]({{ site.baseurl }}/components/button-group/) to step forward and backward through a form.
```